Libs/MRML/Logic/vtkMRMLApplicationLogic.h

```diff
@@ -289,6 +291,15 @@ class VTK_MRML_LOGIC_EXPORT vtkMRMLApplicationLogic
   /// Font file path is set to the one specified in FontFileName property in this object.
   void UseCustomFontFile(vtkTextProperty* textProperty);
 
+  // Translation function for logic classes
+  std::string Translate(const char *context, const char *sourceText, const char *disambiguation = nullptr, int n = -1);
+
+  // Get the object used for string translation in logic classes
```

Comment:
Consider grouping (using `/// @{` and `/// }@`) the Set/Get like done elsewhere in the file.